meson.build

```diff
@@ -93,6 +93,10 @@ if openssl_dep.found()
            description: 'OpenSSL/LibreSSL API version @0@'.format(api_version))
 endif
 
+keyutils_dep = dependency('libkeyutils',
+                          required : get_option('keyutils'))
```

Comment:
We should the same approach as we have done with libdbus. Introduce a meson config and make the dependency really optional. Unfortunately, the required argument of dependency doesn't allow to drop the dependency completely.

Reply:
I don't mind. Being as I am no expert on meson I'd go with whatever the recommendation is.

Reply:
I got a beating for introducing the libdbus dependency. So I am a bit more careful what is necessary :)

Reply:
Updated the dependencies; do check whether it's okay now.